### PR TITLE
Feature/basic information form

### PR DESCRIPTION
**Pull request by sunilbeis**

### CODE REVIEW

File: INZFS.MVC/Controllers/FundApplicationController.cs [hunk 1]
```diff
@@ -87,38 +160,46 @@ public FundApplicationController(IContentManager contentManager, IContentDefinit
...
+            query = query.With<ContentItemIndex>(x => x.ContentType == contentType);
+            query = query.With<ContentItemIndex>(x => x.Published);
+            query = query.With<ContentItemIndex>(x => x.Author == User.Identity.Name);
 
-            var model = await _contentItemDisplayManager.BuildEditorAsync(contentItem, _updateModelAccessor.ModelUpdater, true);
+            var records = await query.CountAsync();
+            if(records > 0)
+            {
+                var existingContentItem = await query.FirstOrDefaultAsync();
+                //return RedirectToAction("Edit", new { contentItemId = existingContentItem.ContentItemId });
```
Comment (lorenzoAtBEIS) [inline INZFS.MVC/Controllers/FundApplicationController.cs:179]: Maybe get rid of this comment

File: INZFS.MVC/Models/PersonPart.cs [hunk 2]
```diff
@@ -0,0 +1,44 @@
+/*
```
Comment (lorenzoAtBEIS) [inline INZFS.MVC/Models/PersonPart.cs:1]: Get rid?

File: INZFS.MVC/Startup.cs [hunk 3]
```diff
@@ -1,56 +1,64 @@
...
-                areaName: "INZFS.MVC",
-                pattern: "FundApplication/Index",
-                defaults: new { controller = "FundApplication", action = "Index" }
+            services.AddTagHelpers<AddClassTagHelper>();
+            services.AddTagHelpers<ValidationMessageTagHelper>();
+            services.AddTagHelpers<ValidationHighLighterTagHelper>();
+            
 
-                );
+            services.AddContentPart<PersonPart>()
```
Comment (lorenzoAtBEIS) [inline INZFS.MVC/Startup.cs:30]: Get rid

File: INZFS.MVC/ViewModels/PersonPartViewModel.cs [hunk 4]
```diff
@@ -0,0 +1,44 @@
+using INZFS.MVC;
```
Comment (lorenzoAtBEIS) [inline INZFS.MVC/ViewModels/PersonPartViewModel.cs:1]: Get rid

File: INZFS.MVC/Views/PersonPart.Edit.cshtml [hunk 5]
```diff
@@ -0,0 +1,59 @@
+@* As you can see the model here is the view model object this time not a ShapeViewModel<T>. *@
```
Comment (lorenzoAtBEIS) [inline INZFS.MVC/Views/PersonPart.Edit.cshtml:1]: Get rid

File: INZFS.MVC/Views/PersonPart.Summary.cshtml [hunk 6]
```diff
@@ -0,0 +1,5 @@
+﻿@using INZFS.MVC.ViewModels
```
Comment (lorenzoAtBEIS) [inline INZFS.MVC/Views/PersonPart.Summary.cshtml:1]: Get rid

File: INZFS.MVC/Views/PersonPart.SummaryAdmin.cshtml [hunk 7]
```diff
@@ -0,0 +1,5 @@
+﻿@using INZFS.MVC.ViewModels
```
Comment (lorenzoAtBEIS) [inline INZFS.MVC/Views/PersonPart.SummaryAdmin.cshtml:1]: Get rid of

File: INZFS.MVC/Views/PersonPart.cshtml [hunk 8]
```diff
@@ -0,0 +1,13 @@
+@using INZFS.MVC.ViewModels
```
Comment (lorenzoAtBEIS) [inline INZFS.MVC/Views/PersonPart.cshtml:1]: Get rid of

File: INZFS/App_Data/Sites/Default/ContentDefinition.json [hunk 9]
```diff
@@ -443,6 +443,108 @@
...
+          "Listable": true,
+          "Draftable": true,
+          "Versionable": true,
+          "Securable": true
+        },
+        "FullTextAspectSettings": {}
+      },
+      "ContentTypePartDefinitionRecords": [
+        {
+          "PartName": "PersonPart",
```
Comment (lorenzoAtBEIS) [inline INZFS/App_Data/Sites/Default/ContentDefinition.json:462]: Get rid of this part

File: INZFS/App_Data/Sites/Default/ContentDefinition.json [hunk 10]
```diff
@@ -949,7 +1051,31 @@
       "Name": "County",
       "Settings": {},
       "ContentPartFieldDefinitionRecords": []
+    },
+    {
+      "Name": "PersonPart",
```
Comment (lorenzoAtBEIS) [inline INZFS/App_Data/Sites/Default/ContentDefinition.json:1056]: Get rid

**Review by lorenzoAtBEIS — COMMENTED** [review]
Hi Sunil could you delete the refs to person model and remove comments, otherwise looks good

File: INZFS/App_Data/Sites/Default/ContentDefinition.json (hunk 9 above)
Comment (GlenHani) [reply INZFS/App_Data/Sites/Default/ContentDefinition.json:462]: Resolved

File: INZFS/App_Data/Sites/Default/ContentDefinition.json (hunk 10 above)
Comment (GlenHani) [reply INZFS/App_Data/Sites/Default/ContentDefinition.json:1056]: resolved

**Review by lorenzoAtBEIS — APPROVED** [review]
Review Completed

**Pull request merged**